CHANGELOG.rst

```diff
@@ -18,57 +18,57 @@
   Importing signals should now be imported from ``django_amazon_ses`` instead
   of ``django_amazon_ses.backends.boto``.
 
-0.3.2
-=====
+0.3.2 - 2017-11-15
+==================
 
 - Fix Travis CI deployment process to account for wheels.
 - Ensure that only one element of the build matrix publishes to PyPI.
 
-0.3.1
-=====
+0.3.1 - 2017-11-15
+==================
 
 - Fix `tox` installation of test dependencies.
 - Add `pip` cache to Travis CI configuration.
 - Include license file in wheel package.
 
-0.3.0
------
+0.3.0 - 2017-03-30
+==================
 
 - Officially support Python 3.x.
 - Use a more sophisticated matrix build process to test Django compatibility.
 - Add support for ``DJANGO_AMAZON_SES_REGION`` setting.
 
-0.2.2
------
+0.2.2 - 2017-01-05
+==================
 
 - Functionally identical to ``0.2.0-0.2.1``, but includes a reStructuredText formatting change to the ``README`` for PyPi compatibility.
 
-0.2.1
------
+0.2.1 - 2017-01-05
+==================
 
 - Functionally identical to ``0.2.0``, but includes a reStructuredText formatting change to the ``README`` for PyPi compatibility.
 
-0.2.0
------
+0.2.0 - 2017-01-05
+==================
 
 - Add support for `pre_send` and `post_send` signals for email messages.
 
-0.1.3
------
+0.1.3 - 2016-03-23
+==================
 
 - Update PyPI credentials; functionally identical to ``0.1.0-0.1.2``.
 
-0.1.2
+0.1.2 - 2016-03-23
```

Comment:
Looks like this one, and the one below it, are at different header levels than the others.